action.yml

```diff
@@ -226,7 +226,6 @@ runs:
 
     - name: Get final state of Testing Farm scheduled request
       id: final_state
-      if: ${{ inputs.update_pull_request_status == 'true'}}
```

Comment:
This may not return results url for a request that is still not scheduled. This can happen if update_pull_request_status is set to false, as wait-for-results step is skipped.